numba/tests/test_np_functions.py

```diff
@@ -375,6 +380,38 @@ def check(x_types, x_values):
         x_types = [types.complex64, types.complex128]
         check(x_types, x_values)
 
+    @unittest.skipIf(NumpyVersion(np.__version__) < '1.12.0', "Numpy bug")
```

Comment:
```suggestion
    @unittest.skipUnless(NumpyVersion(np.__version__) < '1.12.0', "NumPy Unsupported")
```

Reply:
(and fix up the version check for NumPy as per the above, also line wrap to 80 chars!)